modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.tsx

```diff
@@ -252,7 +252,7 @@ export default function EditNotificationTemplate({
 		objectDefinitionId: 0,
 		recipients: recipientInitialValue,
 
-		...(!Liferay.FeatureFlags['LPS-162133'] && {
+		...(Liferay.FeatureFlags['LPS-162133'] && {
```

Comment:
Nara, aqui pelo q eu entendi q Feliphe me passou, com a FF, deveria ser assim:

```
"recipients": [
     {
       "bcc": "test@liferay.com.br",
       "cc": "another.email@liferay.com.br",
       "from": "test@liferay.com",
       "fromName": {"en_US":"Liferay", "pt_BR":"Liferay Brazil"},
       "to":{"en_US":"test@liferay.com", "pt_BR":"test@liferay.com.br"}
     }
   ]
```

Aí coloquei essa negacao para poder ver se n tiver com a FF, faz como era antes, que não fica dentro de recipients.

Reply:
Thank you, Aline :) I'll update that.

Reply:
I'll just caught two other places where I needed to adjust the check of the Feature Flag. But I can't make the adjustment you mentioned above because it wouldn't be possible to save a notification template with CC and BCC informations. I think you got confused in that case.